open-craft/opencraft
# BB-1970: Connect to backend

This PR connects the new Ocim frontend to the v2 backend APIs and adds a few missing features to the registration form.

**Testing instructions:**
1. Run the backend and frontend from this branch.
2. Go through the registration form and complete a registration.
3. Check that the user was created.
4. Click on the email validation links (they are printed on the backend console output).
5. Check that the instance is created and starts provisioning.
6. Check that the field validation is working.
6. Try to break the form: start on a wrong page (such as `/registration/instance`).

**Reviewer:**
